tools/gromacs/trj.xml

```diff
@@ -11,18 +11,28 @@
             </when>
         </xml>
     </macros>
-
-    <expand macro="requirements" />
-
+    <expand macro="requirements"/>
     <command detect_errors="exit_code"><![CDATA[
     #if $trj.trj_op == 'trjcat':
         mkdir trajs &&
-        #for $value, $file in enumerate($trj_input):
-            ln -s '$file' trajs/traj_${value}.${file.ext} &&
+        #set $file_count = 0  ## sets 1st input to 0
+        #for $file_count, $file in enumerate($trj_input):  ## enumerates each input file, starting from 0
+            ln -s '$file' trajs/traj_${file_count}.${file.ext} &&
```

Comment:
You could also create here an array with #set and simply you for loop below and get rid of file_count

Reply:
I see. Well the thing is, we did it this way because we're keeping the original 'multiple select' input, to ensure backwards compatibility if anyone runs an old history with the new version of the tool. But we also added a repeat block after this original input, to allow more options for input trajectories. In order to keep the numbering constant, we had to set 'file_count' at the beginning.

Reply:
What I was suggesting is really just a simplification of cheetah with no change to UX or backward combat.

Reply:
Ok. appreciate the suggestion!

Reply:
Surely using a collection as input preserves order?

Reply:
It seems it doesn't - I even wrote that in the help section - and I can't remember for sure 2 years after the tool was written why not :(

I think it was simply because the alphanumeric sorting messes up after 10 trajectories, i.e. `traj_2` will come after `traj_10`. In which case the correct solution would be to add padding zeroes, but I couldn't work out how to do that in cheetah at the time.